egs2/TEMPLATE/asr1/asr.sh

```diff
@@ -906,6 +919,7 @@ if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ] && ! [[ " ${skip_stages} " =~ [
             --model_type="${bpemode}" \
             --model_prefix="${bpeprefix}" \
             --character_coverage=${bpe_char_cover} \
+            --treat_whitespace_as_suffix=${usesuffixbpe} \
```

Comment:
```suggestion
```

Reply:
Instead, you may add the following above
```
if [ -n "${suffixbpe}" ]; then
    _opts_spm+="--treat_whitespace_as_suffix=${usesuffixbpe} "
fi
```

Reply:
This one (removing the existing one) would break compatibility. Please follow @simpleoier's suggestion.